src/main/java/com/iexec/commons/containers/SgxDriverMode.java

```diff
@@ -18,7 +18,7 @@
 
 import lombok.Getter;
 
-import javax.annotation.Nonnull;
+import jakarta.annotation.Nonnull;
```

Comment:
This looks like a manual modification. If you remove the line and let the IDE propose a new import, this line will be moved elsewhere. Can we do this to stay compliant with imports organization in Java source files ?